datadog_cluster_agent/README.md

```diff
@@ -0,0 +1,59 @@
+# Datadog Cluster Agent Integration
+
+## Overview
+
+This check monitors the [Datadog Cluster Agent][1].
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+The `datadog_cluster_agent` check is included in the [Datadog Agent][2] package.
+No additional installation is needed on your server.
+
+### Configuration
+
+1. Edit the `datadog_cluster_agent.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your `datadog_cluster_agent` performance data. See the [sample datadog_cluster_agent.d/conf.yaml][3] for all available configuration options.
```

Comment:
Since this is K8s, should we document logs? https://github.com/DataDog/integrations-core/blob/3375b04d162dda04de08805bcfb2cd675b2175e5/kube_scheduler/README.md#log-collection